ipatests/prci_definitions/prci_checker.py

```diff
@@ -391,7 +388,7 @@ def main():
     jobs_spec, supported_classes, f_fixed_jobs = process_spec_file(args.spec)
 
     if args.file:
-        result = process_def_file(args.file, jobs_spec, supported_classes)
+        result = process_def_file(args.file, jobs_spec, supported_classes)[0]
```

Comment:
If I understood you correctly, yes, this behaviour is expected.
When you check only a single definition file the number of jobs are not tested/compared to other files. Since you added a new job to nightly_latest.yaml but not to the rest (those defined in prci_jobs_spec.yaml), the general check will fail.

Reply:
Ok makes sense.